pkgs/native_assets_cli/lib/native_assets_cli.dart

```diff
@@ -10,6 +10,7 @@ export 'src/api/build.dart' show build;
 export 'src/api/builder.dart' show Builder;
 export 'src/api/link.dart' show link;
 export 'src/api/linker.dart' show Linker;
+export 'src/code_assets/os.dart' show OS;
```

Comment:
I think we should not be exporting this from here, it's now a `code_assets.dart` thing that people should get via `import packag:native_assets_cli/code_assets.dart`' (or whatever I called it)

Reply:
Ah yes!